d2bs/kolbot/default.dbj

```diff
@@ -63,6 +65,18 @@ function main() {
 
 	// don't load default for mules
 	if (getScript("D2BotMule.dbj")) {
+		while(true) {
+			try {
+				if (me.area === 2) {
```

Comment:
why is this checking for area == 2?

Reply:
As explained in our discord chat, it is a safety check for the mule to protect against a random walk and dying out of act 1.  I tested your recommended change to `if (me.area !== 1) {` and in the event the character is in another area (for example pandemonium fortress (act4 town) the bot will still exit instead of taking the waypoint to act 1. 

Additionally, after forcing crashes to create a random walks the only time the random walk outside of town was in Act 1. The random walks in that occured in Act 2 were the bot moving to either the the corner behind Jerhyn's Castle or entering the Atma's tavern.

Reply:
If you want it to work only when it's in a town you can do me.inTown check instead